### PR TITLE
Add epp malop v2.0.1

### PR DESCRIPTION
**Pull request by suraj-metron**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Cybereason/Integrations/Cybereason/Cybereason.py [hunk 1]
```diff
@@ -26,6 +26,7 @@
 FETCH_TIME = demisto.params().get('fetch_time', FETCH_TIME_DEFAULT)
 FETCH_TIME = FETCH_TIME if FETCH_TIME and FETCH_TIME.strip() else FETCH_TIME_DEFAULT
 FETCH_BY = demisto.params().get('fetch_by', 'MALOP CREATION TIME')
+IS_EPP_ENABLED = demisto.params().get('enable_epp_poll', False)
```
Comment (ilaner) [inline Packs/Cybereason/Integrations/Cybereason/Cybereason.py:29]:
```suggestion
IS_EPP_ENABLED = argToBool(demisto.params().get('enable_epp_poll', False))
```

File: Packs/Cybereason/Integrations/Cybereason/Cybereason.py [hunk 2]
```diff
@@ -540,6 +541,26 @@ def query_malops_command(client: Client, args: dict):
...
+    api_response = client.cybereason_api_call('POST', '/rest/detection/inbox', json_body=json_body)
+    demisto.debug(f"Fetching the length of rest/dectection malops : {len(api_response)}")
+    return api_response
+
+
+def get_non_edr_malop_data(client, start_time):
+    malop_data = poll_malops(client, start_time)
+    non_edr_malop_data = list()
+    for malops in malop_data['malops']:
+        if malops.get('edr') is False:
```
Comment (ilaner) [inline Packs/Cybereason/Integrations/Cybereason/Cybereason.py:556]:
```suggestion
        if not malops.get('edr'):
```

File: Packs/Cybereason/Integrations/Cybereason/Cybereason.py [hunk 3]
```diff
@@ -1450,6 +1471,8 @@ def malop_to_incident(malop: str) -> dict:
         raise ValueError("Cybereason raw response is not valid, malop is not dict")
 
     guid_string = malop.get('guidString', '')
+    if guid_string == "":
```
Comment (ilaner) [inline Packs/Cybereason/Integrations/Cybereason/Cybereason.py:1474]:
```suggestion
    if not guid_string:
```

**Review by ilaner — CHANGES_REQUESTED** [review]
Great job!
Please see the comments and try to reach the necessary coverage in unit tests

**Comment by suraj-metron** [comment]
Thank you @ilaner for your suggestions, we will review and add those.
Also, we will update the Unit test to cover the required percentage of code.

**Review by ilaner — APPROVED** [review]

**Comment by ilaner** [comment]
Great job!

**Pull request merged**